File: Entities/Ventas.cs
namespace Entidades
{
    using System;

    public class Ventas
    {
        private int varIdProducto;
        private int varMes;
        private int varAño;
        private double varCantidad;

        public int IdProducto
        {
            get => 
                this.varIdProducto;
            set => 
                this.varIdProducto = value;
        }

        public int Mes
        {
            get => 
                this.varMes;
            set => 
                this.varMes = value;
        }

        public int Año
        {
            get => 
                this.varAño;
            set => 
                this.varAño = value;
        }

        public double Cantidad
        {
            get => 
                this.varCantidad;
            set => 
                this.varCantidad = value;
        }
    }
}

File: Entidades/Ventas.vb
Public Class Ventas

    Private varIdProducto As Integer
    Public Property IdProducto() As Integer
        Get
            Return varIdProducto
        End Get
        Set(ByVal value As Integer)
            varIdProducto = value
        End Set
    End Property

    Private varMes As Integer
    Public Property Mes() As Integer
        Get
            Return varMes
        End Get
        Set(ByVal value As Integer)
            varMes = value
        End Set
    End Property

    Private varAño As Integer
    Public Property Año() As Integer
        Get
            Return varAño
        End Get
        Set(ByVal value As Integer)
            varAño = value
        End Set
    End Property

    Private varCantidad As Double
    Public Property Cantidad() As Double
        Get
            Return varCantidad
        End Get
        Set(ByVal value As Double)
            varCantidad = value
        End Set
    End Property
End Class
